.github/workflows/unittests.yml

```
@@ -0,0 +1,39 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python Test and Lint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Get full Python version
+        id: full-python-version
+        shell: bash
+        run: echo ::set-output name=version::$(python -c "import sys; print('-'.join(str(v) for v in sys.version_info))")
+      - name: Install dependencies
+        shell: bash
+        run: |
+          python -m pip install -U -r requirements.txt
+      - name: Build and install anonlink
+        shell: bash
+        run: |
+          python -m pip install -e .
+      - name: Test with pytest
+        run: |
+          pytest -q
```

The only extra thing of value that Travis did for us is running the extended test. Maybe we should add that here.
```suggestion
          pytest -q
      - name: Test with extended size 100k
        if: ${{ matrix.python-version == 3.9}}
        env: 
          INCLUDE_100K: 1
        run: |
          pytest -q
```